docs/guides/hello-world.ipynb

```diff
@@ -9,7 +9,16 @@
     "\n",
     "This example contains two parts. You will first create a simple quantum program and run it on a quantum system.  Because actual quantum research requires much more robust programs, in the second section ([Scale to large numbers of qubits](#scale-to-large-numbers-of-qubits)), you will scale the simple program up to utility level.  You can also follow along with the Hello World episode of the Coding with Qiskit 1.0 video series.\n",
     "\n",
-    "<iframe width=\"100%\" height=\"450\" src=\"https://www.youtube.com/embed/93-zLTppFZw?si=whJzB_FrpuorerTv\" title=\"Hello World\" frameborder=\"0\" allow=\"accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture\" referrerpolicy=\"strict-origin-when-cross-origin\" allowfullscreen></iframe>"
+    "<ImageLink\n",
+    "  title=\"Coding with Qiskit 1.x, Episode 3: Hello world\"\n",
+    "  alt=\"Video thumbnail: Dr. Derek Wang stands against a pink background.\"\n",
+    "  href=\"https://youtu.be/93-zLTppFZw\"\n",
+    "  src=\"/images/guides/hello-world/youtube-thumbnail.png\"\n",
+    "/>\n",
+    "\n",
+    "<Admonition title=\"Note\" type=\"note\">\n",
+    "  This video uses the `QiskitRuntimeService.get_backend` method which has since been deprecated. Use [`QiskitRuntimeService.backend`](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#backend) instead.\n",
```

Comment:
```suggestion
    "  This video uses the `QiskitRuntimeService.get_backend` method, which has since been deprecated. Use [`QiskitRuntimeService.backend`](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#backend) instead.\n",
```